src/estimator.h

```diff
@@ -429,6 +449,14 @@ class Estimator : public Component<Estimator, State> {
    *  overlap). */
   Timer timer_;
   std::unique_ptr<std::default_random_engine> rng_;
+
```

Comment:
Variable not needed once code block is moved from `Estimator::VisualMeasInternalTrackerOnly` to `Estimator::tracked_features`.